recipes/propka/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "propka" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://files.pythonhosted.org/packages/a3/7b/e935ce37cbf9ac8fe23ec05f8072fb3a63582b20fb01ef0c24502b5d10ba/PROPKA-3.1.0.tar.gz
+  sha256: ada690067c5ad9ca6be412491380929e5275fa41166aa1fb2c820fcc2dfdb619
+
+build:
+  number: 0
+  entry_points:
+    - propka31 = propka.run:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - setuptools
+
+test:
+  commands:
```

Comment:
the skeleton has also generated one test more ...isn't it?

Reply:
Added!